src/app/exercise/exercise.page/state/exercise-state.service.ts

```diff
@@ -53,7 +51,10 @@ export class ExerciseStateService implements OnDestroy {
   private readonly _originalExercise: Exercise.Exercise = this._exerciseService.getExercise(this._activatedRoute.snapshot.params['id']!);
   private _globalSettings: GlobalExerciseSettings = DEFAULT_EXERCISE_SETTINGS;
   private _adaptiveExercise: AdaptiveExercise = this._adaptiveExerciseService.createAdaptiveExercise(this._originalExercise);
-  private _currentQuestion: Exercise.Question = this.exercise.getQuestion();
+  private _currentQuestion: Exercise.Question = {
+    segments: [],
+  };
```

Comment:
Necessary so the initialization will not be counted as a question (next is being called immediately after anyway). 
Not initializing is not an option since it causes runtime exceptions in this case.